plugins/freeones/main.js

```diff
@@ -200,6 +200,54 @@ module.exports = async ({
     }
   }
 
+  function getMeasurements() {
+    if (isBlacklisted("measurements")) return {};
+    $log("Getting measurements...");
+
+    const braSelector = $('[data-test="p-measurements"] a[href*="%5Bbra%5D"]');
+	let braSize = "";
+	let chestSize = "";
+	let cupSize = "";
+	if (!braSelector) {
+		braSize = "";
+		chestSize = "";
+		cupSize = "";
+	} else {
+		braSize = braSelector.length ? braSelector.attr("href").split("=").slice(-1)[0] : null;
+		chestSize = braSize.substr(0,2);
+		cupSize = braSize.substr(2,1);
+	}
```

Comment:
The first block is redundant, since right above, the initial values are already "";
You can invert the condition to just keep the second block
```suggestion
	if (braSelector) {
		braSize = braSelector.length ? braSelector.attr("href").split("=").slice(-1)[0] : null;
		chestSize = braSize.substr(0,2);
		cupSize = braSize.substr(2,1);
	}
```
You can do the same thing for the other conditions like this